contracts/src/v0.8/tests/MockScrollL2CrossDomainMessenger.sol

```diff
@@ -0,0 +1,55 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.9;
+
+/* Interface Imports */
+import {IL2ScrollMessenger} from "@scroll-tech/contracts/L2/IL2ScrollMessenger.sol";
```

Comment:
Please do not put additional files in the `tests` folder. All project files should be self-contained in their own folder, except for vendored dependencies. So in this case, everything in `l2ep`

Reply:
Sounds good - added the scroll mocks to `l2ep/test/mocks`